charms/istio-pilot/src/charm.py

```diff
@@ -447,10 +447,18 @@ def _get_ingress_data(self, event) -> dict:
         # Get all route data from the ingress interface
         routes = get_routes_from_ingress_interface(ingress_interface, self.app)
 
-        # The app-level data is still visible on a broken relation, but we
+        # The app-level data may still be visible on a broken relation, but we
         # shouldn't be keeping the VirtualService for that related app.
```

Comment:
Do you mind adding a comment here that says that the VS for the removed remote application could still be present in the cluster and what could happen with it?